### PR TITLE
feat: added estimate funding total

### LINKED ISSUE #88: 🚀 Feature: Print the total liftable money above the results table
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

Sometimes there are a lot of potential results and it'd be cool to see the total potential $. Let's add that in!

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by jrachelr**
## PR Checklist

- [x] Addresses an existing open issue: fixes #88 
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Added a sum of the liftable funding estimates above the results table.

### CODE REVIEW

**Comment by jrachelr** [comment]
@JoshuaKGoldberg switched to draft as prettier check failing and I'm not sure why yet!

File: src/components/ResultDisplay.tsx [hunk 1]
```diff
@@ -52,7 +55,7 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 									: b.estimatedMoney - a.estimatedMoney
 								: a.lifted
 								? 1
-								: -1,
```
Comment (JoshuaKGoldberg) [inline src/components/ResultDisplay.tsx]: I think it's this comma deletion that's messing with Prettier. If you run `pnpm run format --write` locally, it should fix it. You might have to `pnpm install` first if your local installed modules are out of date (I'm betting your Prettier is an old version that treats commas differently from the latest one).

File: src/components/ResultDisplay.tsx [hunk 2]
```diff
@@ -35,6 +35,9 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 		<ResultsContainer
 			heading={`${counted(result.length, "Liftable Package")} Found`}
 		>
+			<p className={styles.p}>
+				With a funding estimate of <b>~${summed(result)}</b>
```
Comment (JoshuaKGoldberg) [inline src/components/ResultDisplay.tsx]: [Praise] I love this!

File: src/components/ResultDisplay.tsx [hunk 3]
```diff
@@ -69,3 +72,14 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 function counted(count: number, text: string) {
 	return `${count} ${text}${count === 1 ? "" : "s"}`;
 }
+
+function summed(result: EstimatedPackage[]) {
+	let total = 0;
+	result.forEach((num) => {
+		total += num.estimatedMoney;
+	});
+	const formattedTotal = total.toLocaleString("en-US", {
```
Comment (JoshuaKGoldberg) [inline src/components/ResultDisplay.tsx]: [Praise] The `toLocaleString` is a nice touch! I was worried for usernames like `ljharb`  who have weird fractional numbers that we'd have to do something. This is that thing. 🚀

File: src/components/ResultDisplay.tsx [hunk 4]
```diff
@@ -69,3 +72,14 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 function counted(count: number, text: string) {
 	return `${count} ${text}${count === 1 ? "" : "s"}`;
 }
+
+function summed(result: EstimatedPackage[]) {
+	let total = 0;
+	result.forEach((num) => {
+		total += num.estimatedMoney;
+	});
```
Comment (JoshuaKGoldberg) [inline src/components/ResultDisplay.tsx]: [Suggestion] Not a blocker IMO, but if you want to be all snazzy...

```ts
const total = result.reduce((total, current) => total + current.estimatedMoney, 0);
```

File: src/components/ResultDisplay.tsx [hunk 5]
```diff
@@ -69,3 +72,14 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 function counted(count: number, text: string) {
 	return `${count} ${text}${count === 1 ? "" : "s"}`;
 }
+
+function summed(result: EstimatedPackage[]) {
```
Comment (JoshuaKGoldberg) [inline src/components/ResultDisplay.tsx]: [Suggestion] Not a blocker but maaaaybe a more precise name like `estimations`/`packages` would be easier to reason about for this function?

**Review by JoshuaKGoldberg — APPROVED** [review]
Awesome! Leaving a few comment suggestions but nothing blocking, what do you think?

File: src/components/ResultDisplay.tsx (hunk 1 above)
Comment (jrachelr) [reply src/components/ResultDisplay.tsx]: That was it ty!

File: src/components/ResultDisplay.tsx (hunk 4 above)
Comment (jrachelr) [reply src/components/ResultDisplay.tsx]: All about the snazz😎  Yes, I agree reduce is a more appropriate method here

File: src/components/ResultDisplay.tsx (hunk 5 above)
Comment (jrachelr) [reply src/components/ResultDisplay.tsx]: I agree, I think `sumEstimateFunding` makes it clear what this function is doing

Comment (JoshuaKGoldberg) [reply src/components/ResultDisplay.tsx]: Oh I meant for the function parameter... but yes the function name too. Both!

File: src/components/ResultDisplay.tsx (hunk 4 above)
Comment (jrachelr) [reply src/components/ResultDisplay.tsx]: We could also remove a couple more lines and do:
```
function sumEstimateFunding(result: EstimatedPackage[]) {
	const total = result
		.reduce((total, current) => total + current.estimatedMoney, 0)
		.toLocaleString("en-US", {
			maximumFractionDigits: 0,
		});
```

Comment (JoshuaKGoldberg) [reply src/components/ResultDisplay.tsx]: Jack Nicholson slowly nodding his head yes with a grin

File: src/components/ResultDisplay.tsx (hunk 5 above)
Comment (jrachelr) [reply src/components/ResultDisplay.tsx]: por que no los dos

**Pull request merged**